packages/node_modules/pouchdb-adapter-indexeddb/src/index.js

```diff
@@ -108,23 +107,23 @@ function IndexeddbPouch(dbOpts, callback) {
     return info(metadata, cb);
   });
 
-  api._get = $t(get);
+  api._get = $t(get, [META_STORE, DOC_STORE]); // TODO can be more selective with which store we're opening here
 
   api._bulkDocs = $(function (_, req, opts, callback) {
     bulkDocs(api, req, opts, metadata, dbOpts, idbChanges, callback);
   });
 
   api._allDocs = $t(function (txn, opts, cb) {
     allDocs(txn, metadata, opts, cb);
-  });
+  }, [DOC_STORE]);
 
-  api._getAttachment = $t(getAttachment);
+  api._getAttachment = $t(getAttachment, [META_STORE, DOC_STORE]); // TODO can _local docs have attachments?  Should we decide _which_ store to open here?
```

Comment:
I imagine local docs can have attachments. We should handle it.

Reply:
This should be solved by recent changes in `master` - `indexeddb`'s `_getAttachment()` implementation no longer requires transactions at all.